static/locales/en/design.json

```
@@ -69,10 +69,10 @@
   },
   "devBlock": {
     "heading": "Go&nbsp;from cool <span style='color:#F9555A;'>idea</span> to insanely great <span style='color:#F9555A;'>product</span>",
-    "text": "You can read about our core mission, i.e. making complex websites & web applications, on the <a href='outsourcing-front-end' class='devBlock__link'>relevant page</a>."
+    "text": "You can read about our core mission, i.e. making complex websites & web applications, on the <a href='outsourcing-front-end' class='devBlock__link' data-testid='DesignCases:dev.link'>relevant page</a>."
```

Согласно этой [доке](https://github.com/CSSSR/e2e-tools/blob/master/docs/DATA_TESTID.md), сначала ты указываешь тип, а потом через двоеточие уточняю инфу, т.ё у тебя должно быть вот так: 
```suggestion
    "text": "You can read about our core mission, i.e. making complex websites & web applications, on the <a href='outsourcing-front-end' class='devBlock__link' data-testid='DesignCases:link:dev'>relevant page</a>."
```

---

Названием data-testid занимаются разработчики, по этому я не могу их поправить.

---

>${Контекст}:${тип}:${опциональная уточняющая информация через точку}

`опциональная уточняющая информация через точку` - dev.link не правильно понимаю ?

---

@vkosinov да, верно, но там же в доке написано, что link это тип